static/app/utils/metrics/fields.tsx

```diff
@@ -0,0 +1,59 @@
+import {ColumnType} from '../discover/fields';
+
+export enum SessionMetric {
+  SESSION = 'sentry.sessions.session',
+  SESSION_DURATION = 'sentry.sessions.session.duration',
+  SESSION_ERROR = 'sentry.sessions.session.error',
+  USER = 'sentry.sessions.user',
+}
+
+export enum TransactionMetric {
+  MEASUREMENTS_FP = 'sentry.transactions.measurements.fp',
+  MEASUREMENTS_FCP = 'sentry.transactions.measurements.fcp',
+  MEASUREMENTS_LCP = 'sentry.transactions.measurements.lcp',
+  MEASUREMENTS_FID = 'sentry.transactions.measurements.fid',
+  MEASUREMENTS_CLS = 'sentry.transactions.measurements.cls',
```

Comment:
nit: Would be nice to have prefixes `sentry.sessions` and `sentry.transactions` as a constant.